packages/agent-simulation/src/components/agent-simulation.tsx

```diff
@@ -166,6 +185,23 @@ export const AgentSimulationComponent = ({
     setHasBeenStarted(false);
   };
 
+  const handleChangeSimSpeed = (newSpeed: number) => {
+    const oldSpeed = simSpeedRef.current;
+    log("change-simulation-speed", { oldSpeed, newSpeed });
+
+    simSpeedRef.current = newSpeed;
+    if (visRef.current && visRef.current.ticker) {
+      visRef.current.ticker.maxFPS = newSpeed;
+    }
+
+    setInteractiveState?.(prev => ({
+      ...prev,
+      answerType: "interactive_state",
+      version: 1,
```

Comment:
```suggestion
```
Seems like these shouldn't need to be updated, am I missing something?

Reply:
`prev` won't be defined for a new interactive, so `answerType` and `version` could be undefined. If you remove those lines, you get errors like this:
```
Types of property 'answerType' are incompatible.
    Type '"interactive_state" | undefined' is not assignable to type '"interactive_state"'.
      Type 'undefined' is not assignable to type '"interactive_state"'.
```

I do find that a bit confusing myself, but I'm not sure it's easy to change that. Maybe I should add a comment about it?